init.rb

```diff
@@ -20,4 +20,9 @@
   url 'https://github.com/akiko-pusu/redmine_issue_badge'
   author_url 'http://twitter.com/akiko_pusu'
   requires_redmine :version_or_higher => '2.5.0'
+
+  settings :partial => 'settings/redmine_issue_badge',
+    :default => {
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.
* Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)